smithy-aws-iam-traits/src/main/resources/META-INF/smithy/aws.iam.smithy

```diff
@@ -0,0 +1,136 @@
+$version: "2.0"
+
+namespace aws.iam
+
+/// Provides a custom IAM action name. By default, the action name is the same as the operation name.
+@trait(selector: "operation")
+string actionName
+
+/// A brief description of what granting the user permission to invoke an operation would entail. This description should begin with something similar to 'Enables the user to...' or 'Grants permission to...'
```

Comment:
These should be wrapped and not huge single lines. Does running `smithy format` on the file fix this?

Reply:
No it doesn't. I have updated the comments